rslearn/train/tasks/classification.py

```diff
@@ -120,9 +120,12 @@ def process_inputs(
 
         data = raw_inputs["targets"]
         for feat in data:
-            for property_name, property_value in self.filters:
-                if feat.properties.get(property_name) != property_value:
-                    continue
+            if feat.properties is None:
+                continue
+            if self.filters is not None:
```

Comment:
I can't remember why filters was allowed to be None but I don't see a good reason for it now, so I changed the __init__ argument type / default.